viewer/templates/viewer/help.html

```diff
@@ -138,13 +146,47 @@ <h2>HTML</h2>
         Search "&lt;br&gt;"
       </a>
     </li>
-
     <li class="m-list_item">
       You want to find pages that include a specific image filename.
       <a href="{% url 'index' %}?search_type=html&q=consumer_tools_hero">
         Search "consumer_tools_hero"
       </a>
     </li>
+    <li class="m-list_item">
+      You want to find pages encoded in Traditional Chinese.
+      <a
+        href="{% url 'index' %}?search_type=html&q={{ 'html lang=&quot;zh-Hant&quot;' | urlencode }}">
+        Search "html lang="zh-Hant""
+      </a>
+    </li>
+  </ul>
+</div>
+
+<div class="o-well o-well_alt">
```

Comment:
It feels a little weird that the text wraps even though the well continues to the right. Is that the intended behavior?

<img width="1222" alt="image" src="https://user-images.githubusercontent.com/654645/184199825-11f94382-3654-4ab8-a69b-ecf53166961e.png">

Reply:
I agree it looks a little weird, but this is the standard cf.gov wrapping due to the `max-width` property set.  I can change it if we think that it would look better. The other well on the search page has this as well, but the sentences aren't as long